compiler/rustc_infer/src/infer/outlives/verify.rs

```diff
@@ -288,7 +288,7 @@ impl<'cx, 'tcx> VerifyBoundCx<'cx, 'tcx> {
     ///
     /// It will not, however, work for higher-ranked bounds like:
     ///
-    /// ```compile_fail,E0311
+    /// ```
```

Comment:
If this compiles now then the previous comment is wrong as well (and we should find out why)